creator/ingest_runs/genomic_data_loader.py

```diff
@@ -115,10 +115,14 @@ def load_specimen_harmonized_gf_links(self, genomic_df):
                 CONCEPT.GENOMIC_FILE.TARGET_SERVICE_ID,
             ]
         ]
-        biospec_gen_df[CONCEPT.BIOSPECIMEN.ID] = biospec_gen_df[
-            CONCEPT.BIOSPECIMEN.TARGET_SERVICE_ID
-        ]
-        biospec_gen_df[CONCEPT.BIOSPECIMEN_GENOMIC_FILE.VISIBLE] = True
+        biospec_gen_df = biospec_gen_df.assign(
```

Comment:
If I'm reading this correctly, you are simply renaming this column? Could you use `rename` instead?

Reply:
No, I'm creating a new `CONCEPT.BIOSPECIMEN.ID` column with the same values as the `CONCEPT.BIOSPECIMEN.TARGET_SERVICE_ID` column. The ingest library requires both

Reply:
I see, in that case, does `df.loc[:, CONCEPT.BIOSPECIMEN.ID] = df['CONCEPT.BIOSPECIMEN.TARGET_SERVICE_ID]` work? It seems a lot less convoluted.

Reply:
Yeah you're right

Reply:
That still gives the same warning